app/controllers/api/api_controller.rb

```diff
@@ -118,44 +98,32 @@ def render_expired_token_error
         tag: 'expired-access-token',
         message: 'The provided access token has expired.',
       },
-    }, status: :expired_access_token
+    }, status: EXPIRED_TOKEN_HTTP_CODE
```

Comment:
`:expired_access_token` is not defined, at least not in the Rails version I was in.

Reply:
`:expired_access_token` is defined in [this file](https://github.com/standardnotes/syncing-server/blob/develop/config/initializers/custom_http_status_codes.rb)

Reply:
Oh, hmm, wasn't working for me. Would get 500 error on requests because it wasn't defined. In any case if it's just being used once it's probably better to just have it declared as a const in the same file. I'll delete custom_http_status_codes.rb for now.